src/components/common/CreateUserWizard/index.ts

```diff
@@ -10,3 +9,4 @@ export {
   UserWizardStep2,
 } from './validation';
 export { default as ConfirmEmail } from './ConfirmEmail';
+export { default, ContinueWizard, UserStepTemplate } from './CreateUserWizard';
```

Comment:
Moving the "main" export, i.e. the component entry point (`CreateUserWizard`), to the bottom of the file ensures that all of its dependencies are defined (e.g. `StepUserEmail, StepUserName`), before it loads.